windows-installer/fluent-template.conf

```diff
@@ -34,6 +32,11 @@
   disable_retry_limit
   # Use multiple threads for processing.
   num_threads 8
+  # Use gRPC by default to avoid issues with AWS and SSL certs.
```

Comment:
Why not make it `Use gRPC transport.`, to match the comments in the other configs?

Reply:
Changed.